ur3e: #namespace
# Agent parameters
    control: "force_control"
    xF_tradeoff: [1.,1.,0.,1.,1.,1.]
    n_actions: 2 
    init_pos: [0.29004, -0.96341,  2.21555, -2.82236, -1.57056,  0.28972]
    target_pos: [0.2491 ,  0.18742,  0.04009, -3.14128, -0.00034,  1.57046]
    end_effector_points: [[0.,0.,0.]]

    tgt_pose_indices: [0,1]
    pose_indices: [0,1]
    
    scale_pos: 100

# Reward parameters
    distance_threshold: 0.05
    reward_type: 'l1l2'
    cost:
        l1: 1.0
        l2: 0.1
        alpha: 0.0005
